docs/source/testing-and-debugging.md

```diff
@@ -135,3 +135,35 @@ of the transaction.
         result, _ = await self.call("get_message", return_type=str)
         self.assertEqual(new_message, result)
 ```
+
+#### Accessing Events
+
+If you want to test events, you'll get all the notifications that were emitted on the transaction from the second return
+value of the `call` method. The resulting stack of every notification is a list of stack items, so it's best you unwrap
+them using [neo-mamba unwrapping methods](https://dojo.coz.io/neo3/mamba/api/neo3/api/helpers/unwrap.html). In this
+example, we are testing the "Transfer" event, so we can use the `Nep17TransferEvent` class from `boaconstructor`. If you
+plan to test other events, it's best you also create a class or method that will help you unwrap the stack results. 
+Check out [`Nep17TransferEvent`](https://github.com/CityOfZion/boa-test-constructor/blob/20a45755767b3d919bf7a594cfff6bff78cb72ac/boaconstructor/__init__.py#L302-L314)
+implementation for reference.
```

Comment:
Regarding creating new classes to handle other events, I think it's better to give an example in this file than to just link to the boaconstructor code.
Can you change it to create a class to handle the same Nep17 event? But include a validation for `from` or `to` being None (doesn't need to explain)